Sources/Swift/Core/Tools/ViewCapture/UIRedactBuilder.swift

```diff
@@ -75,6 +75,9 @@ class UIRedactBuilder {
         
         if options.maskAllText {
             redactClasses += [ UILabel.self, UITextView.self, UITextField.self ]
+            // These classes are used by React Native to display text.
+            // We are including them here to avoid leaking text from RN apps with manually initialized sentry-cocoa.
+            redactClasses += ["RCTTextView", "RCTParagraphComponentView"].compactMap(NSClassFromString(_:))
```

Comment:
can we make `redactClasses` a Set here actually (I think it's an array now) to avoid duplicates? Also seems like redactClasses could have strings instead of Classes to make it even lighter, but Classes is also fine I guess

Reply:
See line 108, the classes are mapped to a Set of ObjectIdentifier there.

This entire building of identifiers seems to be quite unclean, but I would propose cleaning it up in another PR.